analyzer/src/funTest/kotlin/managers/GoModFunTest.kt

```diff
@@ -91,4 +91,13 @@ class GoModFunTest : StringSpec({
 
         result.toYaml() should matchExpectedResult(expectedResultFile, definitionFile)
     }
+
+    "Local module dependencies are detected correctly" {
```

Comment:
I'd prefer to add a TODO comment instead and keep the expected result as-is, so that it is clear how it works currently. Ok?

Reply:
I have to admit I don't like that approach, as e.g. in test output you'll see that "Local module dependencies are detected correctly" passed, which looks like the feature was implemented.

Reply:
> so that it is clear how it works currently.

If our goal is to document the current state, that's actually an argument for renaming the test, I believe.

Reply:
I've changed the test name.